src/main/java/org/alfresco/repo/content/AbstractContentReader.java

```diff
@@ -374,7 +398,10 @@ public void contentStreamClosed() throws ContentIOException
 
     /**
      * @see Channels#newInputStream(java.nio.channels.ReadableByteChannel)
+     *
+     * @deprecated The transformations code is being moved out of the codebase and replaced by the new async RenditionService2 or other external libraries.
      */
+    @Deprecated

```

Comment:
We are *not* going to remove this method, so the deprecation here is not correct.
Where an implementation uses deprecated methods, it should not automatically be marked as deprecated.
You do not have to mark as ignored (internally) but you can.